src/main/app/src/assets/i18n/nl.json

```diff
@@ -583,6 +585,7 @@
   "zoeken.filter.toegekend": "Toegekend",
   "zoeken.filter.zaak_status": "Zaak status",
   "zoeken.filter.zaak_resultaat": "Zaak resultaat",
+  "zoeken.filter.zaak_initiator": "Zaak Initiator",
```

Comment:
Zaak initiator (wij zijn geen Duiters ;-)
En heb je deze gemist in de engelse vertalingen? Ik zie hem niet in de diff?